spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new 10, 'Student1'
+  end
+
+  it 'Should return person age' do
+    expect(@person.age).to eq 10
+  end
+
+  it 'Should return person name' do
+    expect(@person.name).to eq 'Student1'
+  end
+
+  # it "Should return person's parent_permission" do
+  # 	expect(@person.parent_permission) .to be true
+  # end
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

- [ ] I would recommend adding test for all the public method you have for the respective class. 

Hint: 
- Add test for `can_use_services?`, `correct_name`, and `add_rental` method.